src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -42,14 +39,15 @@ public class UserAffiliationServiceImpl implements UserAffiliationService {
   @Transactional
   public void createPrimaryUserAffiliation(String eventPayload) {
     String centralTenantId = folioExecutionContext.getTenantId();
-    var pair = getDataFromPayload(eventPayload);
-    if (Objects.isNull(pair)) {
+    var userEvent = parseUserEvent(eventPayload);
+    if (Objects.isNull(userEvent)) {
+      log.warn(EVENT_PAYLOAD_COULD_NOT_BE_PARSED, eventPayload);
       return;
     }
-    var userEvent = pair.getLeft();
-    var consortiaTenant = pair.getRight();
 
     try {
+      var consortiaTenant = tenantService.getByTenantId(userEvent.getTenantId());
```

Comment:
why is this variable called `consortiaTenant` and for some reason, i cant see where it is used.

Reply:
Updated name to 'tenant', it is used later in the method